metrics/models.go

```diff
@@ -203,19 +205,41 @@ type CompleteTestStatus struct {
 
 // TestRunStatus binds a TestRun to a CompleteTestStatus.
 type TestRunStatus struct {
-	Run    *base.TestRun
+	Run    *shared.TestRun
 	Status CompleteTestStatus
 }
 
+// TestRunsMetadata is a struct for metadata derived from a group of TestRun entities.
+type TestRunsMetadata struct {
+	// Deprecated. Store the IDs in TestRunIDs instead.
+	TestRuns   []shared.TestRun `json:"test_runs,omitempty" datastore:"TestRuns,omitempty"`
```

Comment:
Shall we perhaps use `shared.TestRuns` here?

Reply:
Done.